floaters/src/main/java/com/mdg/droiders/floaters/FloatingViewContainer.java

```diff
@@ -21,17 +23,20 @@
      *
      * @param ctx                   The {@link Context} is needed to inflate floating view layout from xml
      * @param isHardwareAccelerated Set false to set {@link View#LAYER_TYPE_SOFTWARE} on the floating view
+     * @param isExpandable          Set true to imply that the floating view may convert into music window on click
      * @see View#getLayerType()
      * @see View#setLayerType(int, android.graphics.Paint)
      * @see View#LAYER_TYPE_SOFTWARE
      * @see View#LAYER_TYPE_HARDWARE
      */
-    FloatingViewContainer(Context ctx, boolean isHardwareAccelerated) {
+    FloatingViewContainer(Context ctx, boolean isHardwareAccelerated, boolean isExpandable) {
```

Comment:
\[Checkstyle\] WARNING: 'ctor def modifier' have incorrect indentation level 4, expected level should be 2\.